code/modules/mining/remote_fob.dm

```diff
@@ -0,0 +1,329 @@
+/////////For remote construction of FOB using a computer on the ship during setup phase
+GLOBAL_LIST_INIT(dropship_lzs, typecacheof(list(/area/shuttle/drop1/lz1, /area/shuttle/drop2/lz2)))
+
+
+/////////////////////////////// Drone Mob
+
+/mob/camera/aiEye/remote/fobdrone
+	name = "Remote Construction Drone"
+	mouse_opacity = MOUSE_OPACITY_OPAQUE
+	icon = 'icons/Marine/remotefob.dmi'
+	icon_state = "drone"
+	var/area/starting_area
+	use_static = FALSE
+	var/turf/spawnloc
+
+/mob/camera/aiEye/remote/fobdrone/Initialize()
+	. = ..()
+	starting_area = get_area(loc)
+
+/mob/camera/aiEye/remote/fobdrone/setLoc(atom/t) //unrestricted movement inside the landing zone
```

Comment:
You'll find a lot of old code using single-letter variables such as this one: `t`
Try to avoid replicating that, and instead give variables more informative names for ease of reading.
For example, here, something like `(atom/target)`